build/BuildSteps.cs

```diff
@@ -107,17 +107,14 @@ public static void AddDistLib()
 
         public static void AddPythonWorker()
         {
-            var pythonWorkerPath = Path.Combine(Settings.OutputDir, "python-worker");
-            Directory.CreateDirectory(pythonWorkerPath);
-            using (var client = new WebClient())
-            {
-                client.DownloadFile(Settings.PythonWorkerUrl, Path.Combine(pythonWorkerPath, "worker.py"));
-                client.DownloadFile(Settings.PythonWorkerConfigUrl, Path.Combine(pythonWorkerPath, "worker.config.json"));
```

Comment:
delete `Settings.PythonWorkerUrl` and `Settings.PythonWorkerConfigUrl` from settings